src/site/content/en/accessible/labels-and-text-alternatives/index.md

```diff
@@ -334,3 +334,105 @@ promotional offers?" like in the VoiceOver example below:
   <img class="w-screenshot" src="./promo-offers.png"
   alt="VoiceOver text output showing 'Receive promotional offers?'">
 </figure>
+
+{% AssessmentCallout 'Use the drop-down below each code sample to check whether the HTML element in the sample has an accessible name.' %}
+{% Tabs 'Samples for knowledge self check' %}
+{% Tab 'sample' %}
+
+```html
+<img src="temp.svg" alt="temp">
+```
+
+{% AssessmentHint 'Does this image have an accessible name?' %}
+**Yes**. Give an image an accessible name by adding an `alt` attribute.
```

Comment:
First impression: It's technicall correct but doesn't seem very descriptive?

Reply:
What do you think would be more helpful?